### PR TITLE
Build and push python minimal image to quay.io/sclorg

### PR DESCRIPTION
**Pull request by phracek**
This pull request adds support for building and pushing s2i-python-container `3.9-minimal` image
to `quay.io/sclorg/python-39-minimal` registry.

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Comment by phracek** [comment]
The tag which is used for `3.9-minimal` image will be used. `-el8` and `3.9-el8`

**Comment by phracek** [comment]
Running tests, but the pull request, does not change any s2i-python-container code :)

[test-all]

File: .github/workflows/build-and-push-minimal.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        run: |
+          # This command returns row with BASE_IMAGE_NAME
+          row=$(grep "BASE_IMAGE_NAME" Makefile)
+          # Return only base image name
+          BASE_IMAGE_ROW=${row/BASE_IMAGE_NAME = /}
+          echo ::set-output name=image_name::$BASE_IMAGE_ROW
+
+      - name: Install python3 packages needed by distgen
+        run: |
+          sudo apt-get update -y && sudo apt-get install -y python3 python3-pip python3-jinja2 python3-yaml python3-distro
```
Comment (frenzymadness) [inline .github/workflows/build-and-push-minimal.yml:30]: I believe that you need only python3 and python3-pip. Jinja, pyyaml and distro (+six) are dependencies of distgen and they are all installed together with distgen from PyPI.

**Review by frenzymadness — COMMENTED** [review]
Just one minor comment from me. Otherwise, it looks very good to me. Thank you for that!

**Comment by phracek** [comment]
Running tests, but the pull request, does not change any s2i-python-container code :)

[test-all]

**Comment by phracek** [comment]
[test-openshift]

**Comment by phracek** [comment]
@frenzymadness Container tests are passing. When the OpenShift tests are finished, we can merge them. We will see if everything works properly or not. Sometimes OpenShift 3 tests are failing at the end, because of OpenShift 3 cluster shutdown. It issue is in progress here: https://github.com/sclorg/container-common-scripts/pull/220

I will rebase container-common-scripts and let's see.

[test-all]

**Comment by phracek** [comment]
[test-openshift]

**Comment by phracek** [comment]
[test-openshift]

**Comment by frenzymadness** [comment]
I'd merge it and test it. What do you think?

**Comment by phracek** [comment]
yes.

**Pull request merged**